lib/openlayers/README.md

```diff
@@ -1,30 +1,3 @@
-# OpenLayers 3
+Manueel gedownload: v6.1.1
 
-[![Travis CI Status](https://secure.travis-ci.org/openlayers/ol3.svg)](http://travis-ci.org/#!/openlayers/ol3)
-[![Coverage Status](https://coveralls.io/repos/openlayers/ol3/badge.svg?branch=master)](https://coveralls.io/r/openlayers/ol3?branch=master)
-
-[OpenLayers 3](http://openlayers.org/) is a high-performance, feature-packed library for creating interactive maps on the web.
-
-## Getting Started
-
-- Download the [latest release](http://openlayers.org/download/)
-- Install with npm: `npm install openlayers`
-- Clone the repo: `git clone git@github.com:openlayers/ol3.git`
-
-## Documentation
-
-Check out the [hosted examples](http://openlayers.org/en/master/examples/), the [workshop](http://openlayers.org/workshop/) or [API docs](http://openlayers.org/en/master/apidoc/).
-
-## Bugs
-
-Please use the [GitHub issue tracker](https://github.com/openlayers/ol3/issues) for all bugs and feature requests. Before creating a new issue, do a quick search to see if the problem has been reported already.
-
-## Contributing
-
-Please see our guide on [contributing](CONTRIBUTING.md) if you're interested in getting involved.
-
-## Community
-
-- Need help? Find it on [stackoverflow using the tag 'openlayers-3'](http://stackoverflow.com/questions/tagged/openlayers-3)
-- Follow [@openlayers](https://twitter.com/openlayers) on Twitter
-- Discuss with openlayers users on IRC in `#openlayers` at `chat.freenode`
+Via npm is er een refactor nodig om de imports goed te zetten. De npm versie distribueert niet de gebuilde versie.
```

Comment:
Zou me niet slecht lijken om daar nu wat tijd in te steken ipv. het in een lib folder te steken.